gnome/celluloid/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           conflicts_build 1.0
```

Comment:
I haven't had coffee yet so could be wrong, but I don't see this PG being used..

Reply:
Thanks. I have dropped `celluloid` from here, since there are objections to having `+libmpv` by default, and otherwise CI won’t pass, if a non-default variant is required.